astro/src/lib/api.js

```diff
@@ -1,7 +1,7 @@
 import { useSanityClient } from "astro-sanity";
 
 export async function getAllPosts() {
-  const query = `*[_type == 'post']{"categoryData": categories[]->{slug, title},author -> {name}, ...} | order(publishedAt desc)`;
+  const query = `*[_type == 'post']{..., "categoryData": categories[]->{slug, title}, author->{name, slug, image, bio}} | order(publishedAt desc)`;
```

Comment:
Besides moving the `...` I remove extra spaces as I noted the examples on their docs don't have spaces and also you don't in the arrow for `categories[]->{slug, title}` for example so it overall seemed the right thing to do ;-)